src/main/java/io/excaliburfrc/robot/subsystems/Climber.java

```diff
@@ -147,11 +151,11 @@ public Command openFullyCommand() {
   }
 
   public Command pullUpRobotCommand() {
-    return left.pullUpCommand().alongWith(right.pullUpCommand());
+    return left.pullUpCommand(right.encoder).alongWith(right.pullUpCommand(left.encoder));
```

Comment:
This is definitely scope creep. I think the correction logic should be handled before splitting to sides.